### ISSUE TITLE
"GeoSvc" component name duplicated in FCCSW

### ISSUE THREAD

**Issue by vvolkl** [cepc/CEPCSW#64]
Now that CEPCSW is part of the of the key4hep-stack, I noticed that both FCCSW and CEPCSW have a component named "GeoSvc". This is a very good example of something that could be moved to common key4hep repository and maintained together. (Also, if setting up both FCCSW and CEPCSW, Gaudi can only load one of the components, and there will most likely be an error)
But I just checked, and the GeoSvc here does some additional (experiment-specific?) things, also using GEAR. I'd propose to rename the experiment specific services, and add a DD4hepGeoSvc to a common key4hep repository, and see if we can refactor some things from the GeoSvc here.

**Comment by mirguest** [comment]
Hi @vvolkl ,
Yes, we can move the common code into key4hep.
I have a short discussion with Weidong in this morning. To quick fix the problem, we decide to add prefix "CEPC" to avoid the duplication in FCCSW.
Tao

**Comment by vvolkl** [comment]
That would work, thanks a lot @mirguest

**Comment by mirguest** [comment]
Hi @vvolkl,
I have a short discussion with other people. They think CEPC prefix is too specific, so their suggestion is to rename it to GeomSvc. If the k4GeoSvc is ready, we could switch to it.
Tao

**Comment by mirguest** [comment]
The source code is replaced using following command:
```bash
$ grep --exclude-dir=".git" GeoSvc -Rl . | xargs sed -i 's^GeoSvc^GeomSvc^g'
```

**Comment by mirguest** [comment]
The file names are replaced with following command:
```bash
$ git mv Detector/GeoSvc/src/Geo{,m}Svc.h
$ git mv Detector/GeoSvc/src/Geo{,m}Svc.cpp
$ git mv Detector/Geo{,m}Svc
$ git mv Detector/DetInterface/DetInterface/IGeo{,m}Svc.h

```

**Comment by mirguest** [comment]
Hi @vvolkl , I have merged the PR #66. Could you test again? 
BTW: how should we start the development of k4GeoSvc?
Thank you.
Tao
